app/views/auctions/multi_bid/_win_header.html.erb

```diff
@@ -27,7 +27,7 @@
     </div>
   </div>
   <% end %>
-<% elsif auction.available? && auction.bids? && current_user && auction.user_is_bidder?(current_user) %>
+<% elsif auction.available? && auction.bids? && current_user && auction.user_is_bidder? %>
```

Comment:
I think I want to look at cleaning up the logic in this partial in a future revision because this is kinda ugly, right?